content/22.vaccines-trad.md

```diff
@@ -0,0 +1,507 @@
+As of {{owid_most_recent_date}}, approximately {{owid_total_vaccinations}} vaccine doses have been administered in at least {{owid_total_countries}} countries worldwide using {{owid_vaccine_counts}} different vaccines [@url:https://ourworldindata.org/coronavirus]. 
+The daily global vaccination rate is currently {{owid_daily_rate}}.
+
+However, the distribution of these doses is not uniform around the globe.
+Vaccination rates are lower in South America (3.1 per 100), Asia (1.9 per 100), Africa (0.3 per 100), and Oceania (0.1 per 100) than in North America, Europe, and Australia [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html].<!--update-->
+Vaccine production and distribution varies from region to region and seems to depend on the availability of the vaccines and potentially a country's resources and wealth [@url:https://www.nytimes.com/2020/12/25/business/coronavirus-vaccines-global-economy.html].
+One effort to reduce these disparities is the COVID-19 Vaccines Global Access (COVAX) Facility, a multilateral initiative as part of the Access to COVID-19 Tools (ACT) Accelerator coordinated by the WHO, Gavi The Vaccine Alliance, and the Coalition for Epidemic Preparedness Innovations (CEPI), the latter two of which are supported by the Bill and Melinda Gates Foundation.
+Their intention is to accelerate the development of COVID-19 vaccines, diagnostics, and therapeutics and to ensure the equitable distribution of vaccines to low- and middle-income countries [@doi:10.1111/1468-0009.12503; @url:https://www.who.int/publications/m/item/fair-allocation-mechanism-for-covid-19-vaccines-through-the-covax-facility].
+COVAX invested in several vaccine programs to ensure they would have access to successful vaccine candidates [@doi:10.1126/science.369.6503.489].
+
+Additionally, the vaccine technologies available differ widely around the globe.
+As we review elsewhere, wealthier nations have invested heavily in mRNA and DNA vaccines.
+In contrast, as we describe above, many countries outside of Europe and North America have developed highly effective vaccines using more traditional approaches.
+<!--@RLordan, let's touch base about how to finalize this!-->
+
+#### Asia
+
+The Asian nations of China and India have played a major role as COVID-19 vaccination developers and providers.
+<!--Add brief description of China's number of vaccines, % of population vaccinated, exports-->
+India has vaccinated approximately 24 million people [@url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution].<!--Automate--> and shipped approximately 58 million COVID-19 vaccines to 66 countries [@url:https://www.mea.gov.in/vaccine-supply.htm].<!--update-->
```

Comment:
Not 1.9bn in India vaccinated?

Reply:
I think this is a very old statistic -- like over a year! I'll pull the correct numbers or automate retrieval.